rpcd/playbooks/roles/kibana/tasks/kibana_pre_install.yml

```diff
@@ -13,50 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Install Kibana apt packages
-  apt:
-    pkg: "{{ item }}"
-    state: latest
-    update_cache: yes
-    cache_valid_time: 600
-  register: install_apt_packages
-  until: install_apt_packages|success
+- name: Add kibana apt-keys
+  apt_key:
+    url: "{{ item.url }}"
+    state: "{{ item.state }}"
+  with_items: kibana_apt_keys
```

Comment:
Looking through the rest of the roles in this repo, none of them have `with_items: {{ var_name }}`. They are all using the same pattern.